package/translate/nl.po

```diff
@@ -1,754 +1,512 @@
-# Translation of resources-monitor in nl
-# Copyright (C) 2024
-# This file is distributed under the same license as the resources-monitor package.
+# Translation of resourcesMonitor-fork in nl
+# Copyright (C) 2021
+# This file is distributed under the same license as the resourcesMonitor-fork package.
 # Heimen Stoffels <vistausss@fastmail.com>, 2021.
 #
 msgid ""
 msgstr ""
-"Project-Id-Version: resources-monitor\n"
-"Report-Msgid-Bugs-To: https://github.com/orblazer/plasma-applet-resources-monitor\n"
-"POT-Creation-Date: 2024-04-05 00:29+0200\n"
-"PO-Revision-Date: 2023-05-31 18:44+0200\n"
-"Last-Translator: orblazer\n"
+"Project-Id-Version: resourcesMonitor-fork\n"
```

Comment:
The header should keep the new name `resources-monitor` and the updated copyright year.

Reply:
Hmm, this seems like a Poedit bug. :\